### PR TITLE
Composer update

### PR DESCRIPTION
**Pull request by tmatsuo**
I'll run the build on jenkins too

### CODE REVIEW

**Comment by bshaffer** [comment]
This is amazing and so much better! LGTM

**Comment by tmatsuo** [comment]
I needed to make another change since Datastore v1beta3 changed the Datetime format.

**Comment by tmatsuo** [comment]
@bshaffer thanks. I will merge this once the tests pass.

File: 2-structured-data/src/DataModel/Datastore.php [hunk 1]
```diff
@@ -56,19 +54,23 @@ public function __construct($datastoreDatasetId)
...
         $query = new \Google_Service_Datastore_Query([
-            'kinds' => [
+            'kind' => [
                 [
                     'name' => 'Book',
                 ],
             ],
-            'order' => 'title',
+            'order' => [
+                "property" => [
```
Comment (broady) [inline 2-structured-data/src/DataModel/Datastore.php]: single quotes

File: 2-structured-data/src/DataModel/Datastore.php [hunk 2]
```diff
@@ -221,9 +230,11 @@ private function bookToProperties(array $book)
         $properties = [];
         foreach ($book as $colName => $colValue) {
             $propName = $this->columns[$colName] . 'Value';
-            $properties[$colName] = [
-                 $propName => $colValue
-            ];
+            if (! empty($colValue)) {
```
Comment (broady) [inline 2-structured-data/src/DataModel/Datastore.php]: `!empty`

(remove space)

File: 5-logging/src/DataModel/Datastore.php [hunk 3]
```diff
@@ -56,19 +54,23 @@ public function __construct($datastoreDatasetId)
...
         $query = new \Google_Service_Datastore_Query([
-            'kinds' => [
+            'kind' => [
                 [
                     'name' => 'Book',
                 ],
             ],
-            'order' => 'title',
+            'order' => [
+                "property" => [
```
Comment (broady) [inline 5-logging/src/DataModel/Datastore.php]: Same here

File: 2-structured-data/src/DataModel/Datastore.php (hunk 1 above)
Comment (tmatsuo) [inline 2-structured-data/src/DataModel/Datastore.php]: good eye, done

File: 2-structured-data/src/DataModel/Datastore.php (hunk 2 above)
Comment (tmatsuo) [inline 2-structured-data/src/DataModel/Datastore.php]: thanks done

**Comment by tmatsuo** [comment]
The jenkins build is back to normal. I'm going to merge this.

**Pull request merged**